qiskit/opflow/primitive_ops/tapered_pauli_sum_op.py

```diff
@@ -0,0 +1,52 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" TaperedPauliSumOp Class """
+
+from typing import Optional, Union
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.circuit import ParameterExpression
+
+from .pauli_sum_op import PauliSumOp
+from ..converters.z2_symmetries import Z2Symmetries
+
+
+class TaperedPauliSumOp(PauliSumOp):
+    """Class for PauliSumOp after tapering"""
+
+    def __init__(
+            self,
+            primitive: SparsePauliOp,
+            z2_symmetries: Optional[Z2Symmetries],
```

Comment:
TaperedPauliSumOp should be constructed with Z2symmetry. Otherwise, it should be usual PauliSumOp.
```suggestion
            z2_symmetries: Z2Symmetries,
```